core/scripts/ocr2vrf/util.go

```diff
@@ -520,3 +532,167 @@ func requestRandomnessCallbackBatch(
 
 	return requestID
 }
+
+func getDKGLatestConfigDetails(e helpers.Environment, dkgAddress string) dkgContract.LatestConfigDetails {
```

Comment:
Organization nit/ this file is becoming quite large, what do you think about moving all this verification-related code to a `verify.go` in the same directory?

Reply:
sounds good :+1: